service/rcv_server.conf

```diff
@@ -1,6 +1,19 @@
 [rcv_server]
-roborio_ip=10.0.0.29
-roborio_port=12345
+roborio_ip=10.0.0.25
```

Comment:
should remember to change it to 10.56.35.2 on the day of the tournament

Reply:
O.K.